core/src/test/scala/unit/kafka/admin/ReassignPartitionsClusterTest.scala

```diff
@@ -76,6 +76,32 @@ class ReassignPartitionsClusterTest extends ZooKeeperTestHarness with Logging {
     super.tearDown()
   }
 
+
+  @Test
+  def testHw(): Unit = {
```

Comment:
nit: can we come up with a better name?

Reply:
Yeah I forgot to change it before uploading the patch. It is changed to `testHwAfterPartitionReassignment` now.